src/grid/grid.directive.spec.ts

```diff
@@ -0,0 +1,77 @@
+import { async, TestBed } from "@angular/core/testing";
+import { By } from "@angular/platform-browser";
+import { TestGridComponent } from "./grid-test.component";
+import { ColumnDirective, GridDirective, RowDirective } from "./grid.directive";
+
+describe("Grid Directive", () => {
+	beforeEach(() => {
+		TestBed.configureTestingModule({
+			declarations: [
+				TestGridComponent,
+				GridDirective,
+				RowDirective,
+				ColumnDirective
+			]
+		});
+	});
+
+	it("should render a grid", async(() => {
+		TestBed.overrideComponent(TestGridComponent, {
+			set: {
+				template: "<div ibmGrid class='test-directive'></div>"
```

Comment:
What is the class used for here?

Reply:
Is used to test that you can add a custom class